cmd/diff-main.go

```diff
@@ -49,21 +49,20 @@ FLAGS:
   {{range .VisibleFlags}}{{.}}
   {{end}}
 DESCRIPTION:
-  Diff only lists missing objects or objects with size differences. It *DOES NOT* compare contents. 
-  i.e. objects of same name and size, but differing in content are not noticed.
+  Diff only calculates differences in object name, size and time. It *DOES NOT*
+  compare the differences in object content. 
```

Comment:
> Diff only calculates differences in object name, size and time. It *DOES NOT* compare the differences in object content.

How about `Diff only calculates differences in object name, size and last modified time. It *DOES NOT* compare  object content.`

Reply:
I've also suggested the same, but @abperiasamy thinks `time` is good enough. 
I'll make the other change (`It *DOES NOT* compare object content.`).